tests/features/targets/performance.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Performance and Consistence adding and removing targets
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+    Admin has issued a Token with scopes 'write:targets', 'delete:targets' and "read:tasks"
+
+    Background:
+        Given admin provided the token to the API requester
+        And the API requester has generated the 'headers' with the token
```

Comment:
These small interactions, implementing different FT for different features, improve us over time. 😄 

I used the `Background` for performance tests here because of this "Feature". We don't want to describe to the user what is required to see the performance.
We need to evaluate the other functional features to see the appropriate steps to demonstrate the feature.
For example, for adding/removing targets, the 'headers'  is a process requirement.
In each case, we need to review and see what is relevant to be described in the

Reply:
Added an issue about this: https://github.com/vmware/repository-service-tuf/issues/199